Upcoming release/c8097ed3-15e7-4941-8eb1-7bed1fadef9d.md

```diff
@@ -80,14 +81,16 @@ To search by products, send the request:
                         "name_asc",
                         "name_desc",
                         "price_asc",
-                        "price_desc"
+                        "price_desc",
+                        "popularity"
```

Comment:
@lenadoc the same, it was not integrated into b2c + requires additional project feature integration. Better to remove?

Reply:
Same as above. If it is not integrated in any of the demo shops and requires additional effort from projects, it's better to remove it

Reply:
Same here, @olehnovatskyi-spryker please add the note "This is valid for Master Suite only and has not been integrated into B2B/B2C Suites yet."